src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -219,9 +233,41 @@ public static class PrereleaseCompatibilityTransformer
             return transformer.SarifLog;
         }
 
-        private static bool ApplyRtm2and3Changes(JObject sarifLog)
+        private static bool ApplyRtm5Changes(JObject sarifLog)
```

Comment:
>JObject [](start = 45, length = 7)

Just a thought for future consideration: Loading JObject takes up more memory than System.Text.Document.JDocument. I did some perf profiling comparing the two in the past. For example, a 26MB json file, JObject allocates 393MB compared to only 54MB from JDocument.

Reply:
+1

---
In reply to: [382840623](https://github.com/microsoft/sarif-sdk/pull/1789#discussion_r382840623) [](ancestors = 382840623)